src/diffpy/srmise/tests/test_dataclusters.py

```diff
@@ -32,3 +33,35 @@ def test___eq__():
             print(f"not-equal test failed on {attr_key}")
             assert not expected == actual
         attributes.update({attr_key: reset})
+
+
+# For reset clusters test, we have two test cases:
+# Precondition: DataClusters object should be a valid object.
+# Case (1): x and y are non-empty with positive res, reset_clusters would reset clusters to largest y arg
```

Comment:
it is not clear to me what this reset method is doing.  These tests don't seem to be testing anything.